helm/basex-validator/values.yaml

```diff
@@ -11,3 +11,4 @@ basexValidator:
     pullPolicy: Always
   port: 8984
   replicaCount: 1
+  maxUnavailable: 0
```

Comment:
I thought quite hard about what the default should be actually, and I'm glad you commented as the behaviour I intended for the defaults was that it would happily terminate and replace the pod, which this wont do! Doh, I'll update it to be 1.